db/migrate/20150515041839_create_activities.rb

```diff
@@ -0,0 +1,23 @@
+# Migration responsible for creating a table with activities
+class CreateActivities < ActiveRecord::Migration
+  # Create table
+  def self.up
```

Comment:
migration can ignore length. I set it in the .rubocop.yml file but Hound ignores the file.